.travis.yml

```diff
@@ -35,6 +35,7 @@ addons:
 install:
   - wget https://github.com/KhronosGroup/SPIRV-LLVM-Translator/releases/download/dev-build/SPIRV-LLVM-Translator-dev-build-linux-Release.zip -O /tmp/SPIRV-LLVM-Translator-dev-build-linux-${BUILD_TYPE}.zip
   - unzip /tmp/SPIRV-LLVM-Translator-dev-build-linux-${BUILD_TYPE}.zip -d spirv-llvm-translator
+  - sudo ln -s /usr/lib/llvm-10/lib/libclang-cpp.so.10 /usr/lib/llvm-10/lib/libclang-cpp.so
```

Comment:
This is a work-around for a strange dependency between `libclang-10-dev` and `libclang-cpp10-dev`. Normally we should just install them both, as it is done on the master(ocl-open-110) branch. But for some reason (a bug?) versions of these packages from `apt.llvm.org` remove each other when installed via `apt get`.  I hope the packages can be installed without problems from official ubuntu repositories.